server/pbenchinacan/pbench-server-only.yml

```diff
@@ -0,0 +1,83 @@
+#
+# This configuration is intended to create a brand new (empty) Pbench Server
+# deployment.  The resulting pod consists of three containers: the Pbench Server
+# and its associated Elasticsearch and PostgreSQL database servers.
```

Comment:
The pod actually consists of just the Pbench Server container, relying on external services for Elasticsearch and PostgreSQL.

(Lines 6-7 need patching, as well.)

Reply:
I am planning on removing this in the final version.  I don't see a way to make the kube method deployable in our production environment.

Reply:
Note I still have a commit in the list of commits with this file, but when I squash this down to the final version it'll be removed.